### DISCUSSION TITLE
Advent of Typescript Wrapped

### DISCUSSION THREAD

**Discussion by bautistaaa** [typehero/typehero#1306] [Ideas]
As an open source project I think it would be nice to share the metrics of the event.

Here are a couple ideas I had(feel free to add in thread)

Accounts Created (starting Dec 1 - Dec 25?)
Total Page Views
number of shared on twitter (hashtag stats)
Total number of submissions(maybe)
Most Solved
Most Submissions(successful/unsuccessful)
Least Solved

We could have a bar graph of all the challenges along the x axis to maybe show some data. cool visualizations would be key here and it could have its own route that is SSG'd

**Comment by zack-o-zack** [comment]
Number of badges earned ?
Bronze, Silver, Gold, Platinum and their count.

I think we can add personalized stats as well, if the user is logged in.
But I'm not sure how we can make it sharable.

**Reply by aadamw** [reply]
> I think we can add personalized stats as well, if the user is logged in.
> But I'm not sure how we can make it sharable.

Separate page under `/[username]` routes, which would be accessible from aot page.
We would add personal link to it when user is logged in.
Users would be able to share it on twitter

**Comment by aadamw** [comment]
Top 3 people that posted most with hashtag on twitter?
